CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
+## [2.4.1] - 2017-08-07
```

Comment:
remove the versioning, it is best to leave this to the maintainer as there no guarantee on merge order or release dates.